var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -70,8 +67,7 @@ class Gcc(AutotoolsPackage):
     depends_on("gmp")
     depends_on("mpc", when='@4.5:')
     depends_on("isl", when='@5.0:')
-    depends_on("binutils~libiberty", when='+binutils ~gold')
-    depends_on("binutils~libiberty+gold", when='+binutils +gold')
+    depends_on("binutils~libiberty", when='+binutils')
```

Comment:
Remind me again why we even depend on binutils? Maybe it should be a build deptype??